extensions/amazon-lambda-http/maven-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -3,25 +3,29 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <modelVersion>4.0.0</modelVersion>
-    <groupId>\${groupId}</groupId>
-    <artifactId>\${artifactId}</artifactId>
-    <version>\${version}</version>
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
     <properties>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <surefire-plugin.version>3.0.0-M5</surefire-plugin.version>
-        <maven.compiler.parameters>true</maven.compiler.parameters>
-        <quarkus.version>999-SNAPSHOT</quarkus.version>
+        <assembly-plugin.version>3.1.0</assembly-plugin.version>
```

Comment:
Defining version properties for some plugins but directly setting versions for other plugins seemed inconsistent, so I introduced this property.
Not sure why it has to be 3.1.0, though. The latest version is 3.3.0.